CONTRIBUTING.md

```diff
@@ -793,16 +793,8 @@ Trivial changes, such as spelling fixes, do not need to be signed.
 ### Granted rights and copyright assignment
 
 It is important to note that the DCO is not a license. The license of the
-project – in our case the Apache License – is the license under which the
-contribution is made. However, the DCO in conjunction with the Apache License
-may be considered an alternate CLA.
-
-The existence of section 5 of the Apache License is proof that the Apache
-License is intended to be usable without CLAs. Users need for the code to be
-open-source, with all the legal rights that imply, but it is the open source
-license that provides this. The Apache License provides very generous
-copyright permissions from contributors, and contributors explicitly grant
-patent licenses as well. These rights are granted to everyone.
+project is the license under which the contribution is made. However, the DCO
+in conjunction with the Vector License may be considered an alternate CLA.
```

Comment:
This is not really part of this PR, but I looked at the full text of the CONTRIBUTING file, and the acronym CLA is never spelled out.

Reply:
That's because CLA is a TLA. I'll update that.